src/Sparrow/Platform/Posix/Syscall.cs

```diff
@@ -186,22 +186,22 @@ public static long read(int fd, void* buf, ulong count)
         // pwrite(2)
         //    ssize_t pwrite(int fd, const void *buf, size_t count, off_t offset);
         [DllImport(LIBC_6, SetLastError = true)]
-        public static extern IntPtr pwrite(int fd, IntPtr buf, UIntPtr count, IntPtr offset);
+        public static extern IntPtr pwrite64(int fd, IntPtr buf, UIntPtr count, long offset);
 
         public static long pwrite(int fd, void* buf, ulong count, long offset)
         {
-            return (long)pwrite(fd, (IntPtr)buf, (UIntPtr)count, (IntPtr)offset);
+            return (long)pwrite64(fd, (IntPtr)buf, (UIntPtr)count, offset);
         }
 
 
         // write(2)
         //    ssize_t write(int fd, const void *buf, size_t count);
         [DllImport(LIBC_6, SetLastError = true)]
-        public static extern IntPtr write(int fd, IntPtr buf, UIntPtr count);
+        public static extern IntPtr write(int fd, IntPtr buf, ulong count);
```

Comment:
Is this valid? Not sure here.

Reply:
This works for both RPi and linux-x64  (verified).
However, for 4.0 I wouldn't touch the linux-x64. 
Is `#if ARM` acceptable here?

Reply:
and if you're asking because of the use of write instead of write64, then "count" is size_t according to : http://man7.org/linux/man-pages/man2/write.2.html
So, in 64 bits it is safe to assume it is 64 bits parameter

Reply:
Shouldn't this be `write64` ?

Reply:
it works with both, however it will be write64 in the upcoming "SysCall class and 32 bits" review